src/Purebred.hs

```diff
@@ -236,14 +236,23 @@ guessPackageArgs dir =
     reversedPath = case reverse path of
       ("lib" : xs) -> xs  -- if component is "lib", drop it
       xs -> xs
-    f s
-      | [".cabal", "store"] `isInfixOf` path
-        && ("purebred-" <> versionString <> "-") `isPrefixOf` s =
-          Just s -- cabal newstyle install
-      | otherwise = Nothing
-    packageId = maybe [] (\s -> ["-package-id", s]) (preview _head reversedPath >>= f)
+    isCabalStore = [".cabal", "store"] `isInfixOf` path
```

Comment:
I hope that doesn't stuff up installing it on NixOS. At the moment I don't have an installable package any way, so lets keep this and fix it in case it does.

Reply:
Sure, no worries.